graph/pull.go

```diff
@@ -553,6 +554,8 @@ func (s *TagStore) pullV2Tag(eng *engine.Engine, r *registry.Session, out io.Wri
 
 	}
 
+	out.Write(sf.FormatStatus(repoInfo.CanonicalName+":"+tag, "The image you are pulling has been verified - This is a tech preview, don't rely on it for security yet."))
```

Comment:
Arnaud asked me to look at this and I'm just getting to it now. 
I would suggest the following: "The image you are pulling has been verified. *Important:* image verification is a tech preview feature and should not be relied on to provide security."